Source/ApplicationServices/LibraryCommands.cs

```diff
@@ -197,6 +210,8 @@ private static async Task<List<ImportItem>> scanAccountAsync(ApiExtended apiExte
 
             var dtoItems = await apiExtended.GetLibraryValidatedAsync(libraryOptions, Configuration.Instance.ImportEpisodes);
 
+            archiver?.AddFile($"{DateTime.Now:u} {account}.json", new JObject { { "Account", account.ToString() }, { "DateTime", DateTime.Now.ToString("u") }, {"Items", JArray.FromObject(dtoItems) } });
```

Comment:
This exposes account name to us directly which I try hard not to do. Since it's only in Verbose though, it's fine.

Reply:
The account name and audible user id are members of `Item`, so if we're not scrubbing those then there's no reason to not use the uncensored account in filenames.

Reply:
Nevermind, no they are not. I wouldn't sworn they were in there. Maybe they're in the raw json received from audible but not in the DTO class?

Anyway, I'll use the masked versions of username and then what do you think about writing the zip file on Debug level instead of verbose?

Reply:
Sounds good. I don't mind some PII leaking in Verbose. It shouldn't in other levels though, incl. Debug. This bring up something else: it might be better for debugging to save the raw json; not the DTO.

Reply:
> it might be better for debugging to save the raw json; not the DTO

I fully agree, but we need to change the Api for that. I'm game to do it, but right now this is a bug that that's rendering at least Italian and Indian user's version unusable. I vote just merge and release this for now. These DTOs will be enough for us to get `Plan` data, which is what we need to fix Audible Plus detection for other regions.

Reply:
Agreed

github is doing something very frustrating: it keeps claiming there's only 1 commit and keeps updating it to your latest. Instead of letting me see all the actual commits.

Reply:
Sorry, that's me. I squashed the change to reduce the number of commits because it's still part of [Add better AYCL detection and add verbose library scan logging](https://github.com/rmcrackan/Libation/pull/526/commits/230e014bb11a56be759abc3b0d2507b5bc3503ad). I'll leave it as its own commit next time.

Reply:
Thanks. From my end, github's PR reviewer page has the option to see all changes or select commits. Especially on large PRs it can be helpful to only see recent changes.

Reply:
> it might be better for debugging to save the raw json; not the DTO

I've been brainstorming ways to do this, and everything I've come up with sucks. I thought of:

- Returning a tuple of items and the raw Json
- adding the source JObject as a member of Item
- adding an event to Api that's fired every time Json is read from the Api's HttpClient, and letting callers subscribe to it to handle logging raw responses.
- adding a static setting to Api that instructs it when and where to save raw Json


They all suck for different reasons, so I was wondering if you had any good ideas for how to capture and log raw responses.

Reply:
@rmcrackan 

Thinking about this some more, I'm actually warming too adding the source JObject as a member of Item. Actually I was thinking of keeping the source JObject for all DTOs as a member of DtoBase. Thoughts?

Reply:
I'm not convinced I engineered that part of the audible api correctly when it assumed everything was json. Example: some errors are just strings. Also the api returns no metadata; only data. If I had to redo it, maybe have the response object include the original result json, error info, etc in addition to the actual data payload. That's letting great be the enemy of good though. I think your idea of adding data to that base class is a quick way to accomplish basically the same thing.